app/views/medicaid/applications/show.html.erb

```diff
@@ -81,28 +87,59 @@
 <div class="accordion mb-5">
   <div class="accordion_panel">
     <input type="checkbox" id="chck1" checked>
-    <label class="accordion_panel-label" for="chck1"><h3>Application Request Payload (from Enroll)</h3></label>
+    <label class="accordion_panel-label" for="chck1">
+      <h3>Application Request Payload (from Enroll)</h3>
+    </label>
     <div class="accordion_panel-content">
       <%= raw(ap(@application_request_payload)) %>
     </div>
   </div>
+
   <div class="accordion_panel">
     <input type="checkbox" id="chck2" checked>
-    <label class="accordion_panel-label" for="chck2"><h3>Medicaid Request Payload (to MitC)</h3></label>
+    <label class="accordion_panel-label" for="chck2">
+      <h3>Medicaid Request Payload (to MitC)</h3>
+    </label>
     <div class="accordion_panel-content">
       <%= raw(ap(@medicaid_request_payload)) %>
     </div>
   </div>
+
   <div class="accordion_panel">
     <input type="checkbox" id="chck3" checked>
-    <label class="accordion_panel-label" for="chck3"><h3>Medicaid Response Payload (from MitC)</h3></label>
+    <label class="accordion_panel-label" for="chck3">
+      <h3>Medicaid Response Payload (from MitC)</h3>
+    </label>
     <div class="accordion_panel-content">
       <%= raw(ap(@medicaid_response_payload)) %>
     </div>
   </div>
+
+  <div class='accordion_panel'>
+    <input type='checkbox' id='chck4' checked>
+    <label class='accordion_panel-label' for='chck4'>
+      <h3>Dynamic SLCSP Request Payload (to Enroll)</h3>
+    </label>
+    <div class='accordion_panel-content'>
+      <%= raw(ap(@dynamic_slcsp_request_payload)) %>
+    </div>
+  </div>
+
+  <div class='accordion_panel'>
+    <input type='checkbox' id='chck5' checked>
+    <label class='accordion_panel-label' for='chck5'>
+      <h3>Dynamic SLCSP Response Payload (from Enroll)</h3>
+    </label>
+    <div class='accordion_panel-content'>
+      <%= raw(ap(@dynamic_slcsp_request_payload)) %>
```

Comment:
Is this supposed to be the response payload variable? 

```
@dynamic_slcsp_response_payload
```

Reply:
Good catch!